gemfiles/ruby_3.2.0_contrib.gemfile

```diff
@@ -49,7 +49,7 @@ gem "elasticsearch-transport"
 gem "ethon"
 gem "excon"
 gem "grape"
-gem "graphql", ">= 1.12.0", "< 1.13.0"
+gem "graphql", ">= 2.0"
```

Comment:
Just wondering if these were expected in this changeset.

Reply:
I can explain this one:

1. https://github.com/DataDog/dd-trace-rb/pull/1974 and https://github.com/DataDog/dd-trace-rb/pull/1982 were opened concurrently
2. I merged https://github.com/DataDog/dd-trace-rb/pull/1974 first
3. I suggested Marco not to bother updating the gemfiles/ for 3.2 as anyway they don't run in CircleCI yet (https://github.com/DataDog/dd-trace-rb/pull/1982#discussion_r858432943)
4. The changes showed up here when Marco run the command to update all gemfiles

So this is 100% harmless :)